### PR TITLE
installing babel-plugin-styled-components and lodash

### PR DESCRIPTION
**Pull request by marsinearth**
## Description

some miscellaneous perks by installing babel-plugin-styled-components and lodash

## Related Issues

removed react-spring due to incompatibility with expo. it seems with react-spring installed and being used, javascript building through expo stops at either 18% or 34% and doesn't improve further from there. So removed it.

## Tests

I added the following tests:

• add renderer.act for react-test-renderer based test code to cover useState part.
• add '[Intro] screen useTransition with setInterval test' and put setTimeout for 1500ms so that the first interval has been passed, so that the expected results of the animated text should match with the titleArray's second element.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/whatssub/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: CLA.md [hunk 1]
```diff
@@ -20,6 +20,6 @@ You represent that each of Your Contributions is Your original creation (see sec
...
 
-Should You wish to submit work that is not Your original creation, You may submit it to the Community separately from any Contribution, identifying the complete details of its source and of any license or other restriction (including, but not limited to, related patents, trademarks, and license agreements) of which you are personally aware, and conspicuously marking the work as "Submitted on behalf of a third-party: [named here]".
+Should You wish to submit work that is not Your original creation, You may submit it to the Community separately from any Contribution, identifying the complete details of its source and of any license or other restriction (including, but not limited to, related patents, trademarks, and license agreements) of which you are personally aware, and conspicuously marking the work as "Submitted on behalf of a third-party: [Mars Kim]".
```
Comment (hyochan) [inline CLA.md]: ?? Please revert this

File: package.json [hunk 2]
```diff
@@ -45,12 +47,14 @@
...
     "@types/expo": "^32.0.13",
     "@types/jest": "^24.0.15",
+    "@types/lodash": "^4.14.135",
     "@types/react": "^16.8.22",
     "@types/react-native": "0.57.63",
     "@types/react-test-renderer": "^16.8.2",
     "@types/styled-components": "^4.1.16",
     "@typescript-eslint/parser": "^1.11.0",
     "babel-jest": "^24.8.0",
+    "babel-plugin-styled-components": "^1.10.5",
```
Comment (hyochan) [inline package.json]: Just a question is this necessary?

File: src/components/screen/Intro.tsx [hunk 3]
```diff
@@ -97,15 +109,12 @@ interface IProps {
   navigation?: NavigationScreenProp<NavigationStateRoute<any>>;
 }
 
+export const titleArray = _range(5).map((index:number) =>
```
Comment (hyochan) [inline src/components/screen/Intro.tsx]: Great code!!

File: package.json (hunk 2 above)
Comment (godon019) [reply package.json]: That helps debugging about locating components through DOM elements inspection. Check this [doc](https://www.styled-components.com/docs/tooling#better-debugging).

Comment (hyochan) [reply package.json]: Ah looks great!!

**Comment by hyochan** [comment]
@all-contributors please add @mars-lan for code, test.

**Comment by hyochan** [comment]
@all-contributors please add @marsinearth for code, test.

**Pull request merged**